demo/site-pages/src/theme.ts

```diff
@@ -57,10 +57,11 @@ export const theme = {
     },
     fontFamily: "Arial, sans-serif",
     breakpoints: {
-        xs: createBreakpoint(600),
-        sm: createBreakpoint(900),
-        md: createBreakpoint(1200),
-        lg: createBreakpoint(1600),
+        xs: createBreakpoint(0),
```

Comment:
Do we even need this breakpoint?

Reply:
No, i do not think so. This would not make sense, or what do you think?

Reply:
You're right, I can't think of a use case where it would make sense. Removed: d786b0ada7c4165421625db4b26d9abba96958f5.